data/projects.js

```diff
@@ -182,4 +182,50 @@ exists."
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"]
 	}
+	{
+		"name" : "Debugging and profiling bears",
+		"desc" : "If someone writes a bear he/she needs to test it and comes\
```

Comment:
the first sentence will be shown in the overview, let's make it a good and concise summary